CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ Start OpenSearch and OpenSearch Dashboards. Then refer to the [test execution fi
 E.g if you want to run all plugin tests with security enabled.
 
 ```
-npx cypress run --env SECURITY_ENABLED=true --spec "cypress/integration/plugins/*/*"    
+npx cypress run --env SECURITY_ENABLED=true --spec "cypress/integration/plugins/*/*"
```

Comment:
can we drop these line feed changes?

Reply:
This is from prettier to remove trailing spaces, i followed instructions from [CONTRIBUTING.md](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/CONTRIBUTING.md#formatting)

Reply:
Got it